tests/foreman/api/test_reporttemplates.py

```diff
@@ -369,22 +380,76 @@ def test_negative_create_report_without_name():
 
 
 @pytest.mark.tier2
-@pytest.mark.stubbed
-def test_positive_applied_errata():
+@pytest.mark.rhel_ver_list([7, 8, 9])
+@pytest.mark.no_containers
+def test_positive_applied_errata(
+    module_org, module_cv, module_lce, activation_key, rhel_contenthost, target_sat
+):
     """Generate an Applied Errata report
 
     :id: a4b577db-141e-4871-a42e-e93887464986
 
-    :setup: User with reporting access rights, some host with applied errata
+    :setup: A Host with some applied errata applied errata
 
     :steps:
 
-        1. POST /api/report_templates/:id/generate
+        1. Generate an Applied Errata report
 
     :expectedresults: A report is generated with all applied errata listed
 
     :CaseImportance: Medium
     """
+    ERRATUM_ID = str(settings.repos.yum_6.errata[2])
```

Comment:
Is there any reason for the capitalized name?

Reply:
It's being used in this case as a constant, so python convention